_yaml/data-functions.yml

```diff
@@ -316,6 +343,15 @@ df_date:
         options:  *df_durationUnit
     returns: Date
     description: Adds a given time period to a Date. Returns either a JavaScript Date or Luxon Date, depending on input.
+  - funcName: toDateTime
+    returns: Date
+    description: Converts a JavaScript date to a <a href="https://docs.n8n.io/code/cookbook/luxon/">Luxon date object</a>.
+  - funcName: toFloat
```

Comment:
`toFloat` and `toInt` on Date should be excluded, because `toMillis` and `toSeconds` exist on Luxon and are more clear. Or maybe toMillis and toSeconds should be mentioned here?